Libraries/Text/RCTTextInput.h

```diff
@@ -41,6 +43,8 @@
 @property (nonatomic, copy) RCTDirectEventBlock onContentSizeChange;
 @property (nonatomic, copy) RCTDirectEventBlock onSelectionChange;
 
+@property (readonly, nonatomic, strong) UIFont *font;
```

Comment:
Instad of having real prop, we probably want to have only setter which will redirect the value to `backedTextInputView.font`. (Also we have to add `font` to `BackedTextInputProtocol` if it does not have it already.)